awx/api/views.py

```diff
@@ -2748,17 +2789,14 @@ def post(self, request, *args, **kwargs):
         prompted_fields = _accepted_or_ignored[0]
         ignored_fields.update(_accepted_or_ignored[1])
 
-        for fd, model in (
-                ('credential', Credential),
-                ('vault_credential', Credential),
-                ('inventory', Inventory)):
-            if fd in prompted_fields and prompted_fields[fd] != getattrd(obj, '{}.pk'.format(fd), None):
-                new_res = get_object_or_400(model, pk=get_pk_from_dict(prompted_fields, fd))
-                use_role = getattr(new_res, 'use_role')
-                if request.user not in use_role:
-                    raise PermissionDenied()
-
-        for cred in prompted_fields.get('extra_credentials', []):
+        fd = 'inventory'
+        if fd in prompted_fields and prompted_fields[fd] != getattrd(obj, '{}.pk'.format(fd), None):
+            new_res = get_object_or_400(Inventory, pk=get_pk_from_dict(prompted_fields, fd))
+            use_role = getattr(new_res, 'use_role')
+            if request.user not in use_role:
+                raise PermissionDenied()
```

Comment:
Someone shouldn't get a 403 if they submit a list that's the same list as the JobTemplate credentials list. In this code, I think they would. This would confusingly break the use-case for the API browser, because we automatically insert those values into the entry field (see `update_raw_data`).

If you don't fix that for this merge, could you please file a issue for it?